pkg/kp/rc.go

```diff
@@ -0,0 +1,14 @@
+package kp
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+)
+
+type ReplicationControllerStore interface {
+	Create(manifest pods.Manifest, nodeSelector, podSelector labels.Selector) (string, error)
```

Comment:
Shouldn't there also be a `podLabels` set? You need to know how to label the pod as well as select it.

Probably worth documenting the use of each in the interface as well.